keras/legacy/interfaces.py

```diff
@@ -7,10 +7,12 @@
 def generate_legacy_interface(allowed_positional_args=None,
                               conversions=None,
                               preprocessor=None,
-                              value_conversions=None):
+                              value_conversions=None,
+                              removed_kwargs=None):
```

Comment:
Use instead a preprocessing function to remove args: https://github.com/fchollet/keras/blob/keras-2/keras/legacy/interfaces.py#L9